tasks/10_remove-whitespaces/remove-whitespaces.js

```diff
@@ -0,0 +1,4 @@
+export function trimWhiteSpaces(str) {
+  let trimmedStr = str.replace(/ /g, '');
```

Comment:
Please use const instead of let in this case

Reply:
changed